flask_wtf/_compat.py

```diff
@@ -6,10 +6,12 @@
 if not PY2:
     text_type = str
     string_types = (str,)
+    from collections.abc import Iterable
```

Comment:
Please use the pattern I showed previously. I want to export `_compat.abc`, not `_compat.Iterable`.

Reply:
Sorry, time to quit for today. I hope it is fine now.

I am curious. Is there a technical or an aesthetical reason?

And why didn't you do something like this also?
```
if not PY2:
    import urllib.parse as urlparse
else:
    import urlparse
```